lib/components/textEditor/richText.test.js

```diff
@@ -14,7 +14,7 @@ describe('RichText: ', () => {
     expect(component.getByText('placeholder text')).toBeInTheDocument();
     expect(component.getByTestId('richText').firstChild).toHaveClass('DraftEditor-root');
     expect(component.getByTestId('richText').lastChild).toHaveClass('toolbar');
-    expect(component.getAllByRole('button')).toHaveLength(12);
+    expect(component.getAllByRole('button')).toHaveLength(6);
```

Comment:
Nit: What is this testing? It is not clear. What does it mean to have 6 buttons? If we expect certain buttons, then the test should test that those specific buttons are shown.